StringR.Store/Enum/OrderStatus.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  OrderStatus.swift
+//  StringR.Store
+//
+//  Created by Marcus Christiansen on 18/10/2019.
+//  Copyright © 2019 StringR. All rights reserved.
+//
+
+import Foundation
+
+enum OrderStatus: String, Codable {
+    case RECIEVED = "Recieved"
+    case DONE = "Done"
+    case DELIVERED = "Delivered"
+    case COMPLETE = "Complete"
+}
```

Comment:
Jeg kom lige til at tænke på om vores Enums egentlig ikke skulle være tal. Jeg tror det er langt hurtigere for computeren at evaluere tal end strings :-) 
Så fandt jeg det her på SO: "Comparing primitive numbers is definitely faster than comparing strings because it's just one computer instruction", men det ikke noget vi behøver at kigge på nu :-)